docs-source/content/quickstart/get-images.md

```diff
@@ -38,8 +29,8 @@ and accept the license agreement for the [WebLogic Server image](https://hub.doc
      $ docker pull container-registry.oracle.com/middleware/weblogic:12.2.1.4
      ```
 
-    {{% notice note %}} The WebLogic Server Docker image, `weblogic:12.2.1.3`, has all the necessary patches applied. The WebLogic Server Docker image, `weblogic:12.2.1.4`, does not require any additional patches.
+    {{% notice note %}} The WebLogic Server image, `weblogic:12.2.1.3`, has all the necessary patches applied. The WebLogic Server Docker image, `weblogic:12.2.1.4`, does not require any additional patches.
```

Comment:
The WebLogic Server Docker image, `weblogic:12.2.1.4`, -> The WebLogic Server image,, `weblogic:12.2.1.4`,